### ISSUE TITLE
ACE Interaction Key working locally but not on server

### ISSUE THREAD

**Issue by rbwakefield** [acemod/ACE3#5812]
**Arma 3 Version:** `1.76.143187` (stable)
**CBA Version:** `3.4.1` (stable)
**ACE3 Version:** `3.11.0` (stable)

**Mods:**
```
@ace
@ACEX
@ALiVE
@Ares Mod - Achilles Expansion
@Blastcore Edited (standalone version)
@CBA_A3
@CUP_Terrains_Core
@CUP_Units
@CUP_Vehicles
@CUP_Weapons
@voyagerCompass HUD

```

**Description:**
I built an ALiVE / ACE3 mission locally, and when testing the mission file, the ACE3 interaction key is working properly. When the mission file and required mods were uploaded to our server machine and the mission tested there, ACE interaction key produces no effect - no red circle, nothing.

**Steps to reproduce:**
While working in the editor, the ACE interaction menu worked properly. Testing the mission through self-hosted multiplayer also worked. Upon loading the missions and required mods to our dedicated server machine, the ACE interaction menu stopped working. There is no red dot when the interaction key(s) are pressed. Default keybinds were changed and reset, and a new profile was created. The problem persisted. Additionally, no entries appear in the ACE Options menu (imgur link for screenshot; https://imgur.com/a/3NVrj ).

Thank you for any help / suggestions given!

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer /  Editor (Multiplayer)

**Placed Modules:**

**RPT log file:**
https://pastebin.ca/3939522

**Comment by bux** [comment]
> If you have found an issue with ACE3 please make sure that ACE3 really is the cause of the problem. To do this try to reproduce the issue with using only @CBA_A3 and @ace on a newly created mission.

**Comment by rbwakefield** [comment]
Will do.

**Comment by dedmen** [comment]
Also please post the Client's RPT log if you are having Clientside issues.
Also
>While working in the editor, the ACE interaction menu worked properly. Testing the mission through self-hosted multiplayer also worked.

>**Where did the issue occur?**
> - Dedicated / Self-Hosted Multiplayer / Editor (Multiplayer)

you are contradicting yourself. This is confusing and makes it harder to track down the bug as one would need to ask you which is really true and which is a typo.

**Comment by rbwakefield** [comment]
Sorry for not being more clear. My only excuse is that it was early in the day.

This issue can be closed. I removed all mods from the dedicated machine and began slowly adding them back in, testing after each addition. A process, to be sure, but a welcome one. Eventually all mods were added back in and things were working fine.

The downside is that I wasn't able to sort out where the problem originated from.

Thank you for your comments.
